src/Pyz/Yves/ShopUi/Theme/default/components/molecules/date-time-picker/date-time-picker.ts

```diff
@@ -1,115 +1,71 @@
 import Component from 'ShopUi/models/component';
-import $ from 'jquery/dist/jquery';
-import 'jquery-datetimepicker/build/jquery.datetimepicker.full';
 
 export default class DateTimePicker extends Component {
     protected trigger: HTMLInputElement;
     protected dateFrom: HTMLInputElement;
     protected dateTo: HTMLInputElement;
+    protected datepicker: HTMLInputElement;
 
     protected readyCallback(): void {}
 
     protected init(): void {
-        this.trigger = <HTMLInputElement>this.getElementsByTagName('input')[0];
+        this.trigger = <HTMLInputElement>this.querySelector('input[type="text"]');
```

Comment:
```suggestion
        this.trigger = this.querySelector<HTMLInputElement>('input[type="text"]');
```
Please, fix everywhere